.github/workflows/main.yml

```diff
@@ -0,0 +1,24 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Build with Maven
+        run: mvn --batch-mode --update-snapshots package
+      - run: mkdir staging && cp target/*.jar staging
+
+      - name: Release
```

Comment:
Let's remove this for now, not useful to spread artifacts all over the place without an (automated) install process and no signing. We have dist.metrist.io if we want to distribute stuff outside the stack's regular channels.

Reply:
Removed and moved it to dist.metrist.io